setup.py

```diff
@@ -1,7 +1,7 @@
 from distutils.core import setup
 
-setup(name='pygeotemporal',
-      version='0.1.0',
-      url='https://opensource.ncsa.illinois.edu/bitbucket/projects/GEOD/repos/pygeotemporal/browse',
-      packages=['pygeotemporal'],
+setup(name='pygeostreams',
+      version='0.2.0',
```

Comment:
4.0.0

Reply:
made this change throughout and added myself to docs copyright